package.json

```diff
@@ -46,7 +46,7 @@
     "commander": "~12.1.0"
   },
   "devDependencies": {
-    "@types/jest": "^29.2.6",
+    "@jest/globals": "^29.3.1",
```

Comment:
Can be omitted. It ships with `jest` anyways.

Reply:
We are importing from it, so we should be depending on it. The Jest [Getting Started](https://jestjs.io/docs/getting-started#using-typescript) also shows installing it explicitly.

Reply:
In other words, fine as is!